src/main/java/com/enderio/base/common/block/painted/PaintedSlabBlock.java

```diff
@@ -35,4 +38,15 @@ public Block getPaint(BlockGetter level, BlockPos pos) {
         }
         return IPaintedBlock.super.getPaint(level, pos);
     }
+
+    @Override
+    public ItemStack getCloneItemStack(BlockState state, HitResult target, BlockGetter level, BlockPos pos, Player player) {
+        ItemStack stack = new ItemStack(this);
+        BlockEntity be = level.getBlockEntity(pos);
+        if (be != null) {
+
+            stack.getOrCreateTag().put("BlockEntityTag", be.saveWithoutMetadata());
+        }
+        return stack;
```

Comment:
would it be possible to copy this into a Helper. Also this doesn't work for slabs, as each slab has it's own tag, resulting in a void paint if the top half is selected and both paints being stored in the item for the bottom half, making it unstackable with other obtained versions of this block

Reply:
What would it need to work? Or will it simply not?